src/devtools/devtools.tsx

```diff
@@ -196,6 +197,9 @@ export function ReactQueryDevtools({
     ...otherToggleButtonProps
   } = toggleButtonProps
 
+  // Do not render on the server
+  if (!isMounted()) return null
```

Comment:
why would we check for `isMounted` here? Wouldn't a check for `isServer` be sufficient?

https://github.com/tannerlinsley/react-query/blob/79ad5a9ccbce1486c10e7e364d8fc28bef3fa19f/src/devtools/utils.ts#L8

It would also let us get rid of the comment, because it would be self-explanatory :)

Reply:
That's because doing this can break rehydration. You can read more about it here: https://www.joshwcomeau.com/react/the-perils-of-rehydration/
I actually tried and it breaks on the Next example. A two-pass rendering is safer. (renders `null` on the server, then `null` when rehydrating, then finally the client-side only component).
This way we don't break rehydration.

Reply:
<img width="1249" alt="Screenshot 2021-08-29 at 19 41 56" src="https://user-images.githubusercontent.com/522007/131250723-1901010c-0acd-4ad4-97cf-cf52a867e3be.png">

Reply:
Thanks, today I learned 😅

Reply:
Heh, no problem, always something new to learn! If you every find your website broken in prod but not in dev with super weird styles or DOM, this can be the cause. Like some `div` closing on an other `div` or some crazy stuff like that.